modules/visualization/dev-instance/setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
+
+with open("README.md") as fp:
+    long_description = fp.read()
+
+setuptools.setup(
+    version="0.1.0",
+    description="A short description of the module.",
```

Comment:
Should we populate this with a proper description?

Reply:
Yes...we should.  PR issued